swagger.yaml

```diff
@@ -6235,16 +6296,19 @@ definitions:
   event:
     type: object
     required:
-      - cluster_id
       - severity
       - message
       - event_time
     properties:
+      event_id:
+        type: string
+        format: uuid
```

Comment:
I'm not sure that event-id is needed for identifying a specific event by its ID.
The identifier of the event is its name, and by querying for event name and entity id (cluster/infraenv/host) - we can get the desired event instead of relying on string matching.

Reply:
changed to event_name.
working on the code to populate the values.